pipelines/sdc-responses-dashboard.yml

```diff
@@ -0,0 +1,204 @@
+  source:
+    uri: https://github.com/ONSdigital/ras-deploy.git
+    branch: master
+
+- name: cf-resource-latest
+  type: cf
+  source:
+    api: ((cloudfoundry_api))
+    username: ((cloudfoundry_email))
+    password: ((cloudfoundry_password))
+    organization: rmras
+    space: concourse-latest
+    skip_cert_check: true
+
+- name: cf-resource-preprod
+  type: cf
+  source:
+    api: ((preprod_cloudfoundry_api))
+    username: ((preprod_cloudfoundry_email))
+    password: ((preprod_cloudfoundry_password))
+    organization: rmras
+    space: preprod
+    skip_cert_check: true
+
+- name: cf-cli-resource-preprod
+  type: cf-cli-resource
+  source:
+    api: ((preprod_cloudfoundry_api))
+    username: ((preprod_cloudfoundry_email))
+    password: ((preprod_cloudfoundry_password))
+    org: rmras
+    space: preprod
+    skip_cert_check: true
+
+- name: cf-resource-prod
+  type: cf
+  source:
+    api: ((prod_cloudfoundry_api))
+    username: ((prod_cloudfoundry_email))
+    password: ((prod_cloudfoundry_password))
+    organization: rmras
+    space: prod
+    skip_cert_check: true
+
+- name: cf-cli-resource-prod
+  type: cf-cli-resource
+  source:
+    api: ((prod_cloudfoundry_api))
+    username: ((prod_cloudfoundry_email))
+    password: ((prod_cloudfoundry_password))
+    org: rmras
+    space: prod
+    skip_cert_check: true
+
+- name: notify
+  type: slack-notification
+  source:
+    url: ((slack_webhook))
+
+jobs:
+- name: sdc-responses-dashboard-latest-deploy
+  serial: true
+  plan:
+  - get: sdc-responses-dashboard-source
+    trigger: true
+  - get: ras-deploy
+  - task: run-unit-tests
+    file: ras-deploy/tasks/python-unit-tests.yml
+    input_mapping: { repository-name: sdc-responses-dashboard-source }
+  - put: push-app
+    resource: cf-resource-latest
+    params:
+      current_app_name: sdc-responses-dashboard-latest
+      manifest: sdc-responses-dashboard-source/manifest.yml
+      path: sdc-responses-dashboard-source
+      environment_variables:
+        HOST: 0.0.0.0
+        LOGGING_LEVEL: DEBUG
+        REPORTING_REFRESH_CYCLE_IN_SECONDS: "60"
+        COLLECTION_EXERCISE_URL: http://rm-collection-exercise-service-concourse-latest.apps.devtest.onsclofo.uk
+        SURVEY_URL: http://rm-survey-service-concourse-latest.apps.devtest.onsclofo.uk
+        REPORTING_URL: http://rm-reporting.apps.devtest.onsclofo.uk
+        AUTH_USERNAME: ((latest_security_user_name))
+        AUTH_PASSWORD: ((latest_security_user_password))
+  on_failure:
+    put: notify
+    params:
+      text:  |
+        Devtest Latest space responses-dashboard deploy failed. See build:
+        $ATC_EXTERNAL_URL/teams/$BUILD_TEAM_NAME/pipelines/$BUILD_PIPELINE_NAME/jobs/$BUILD_JOB_NAME/builds/$BUILD_NAME
+
+
+- name: sdc-responses-dashboard-preprod-deploy
+  serial: true
+  plan:
+  - get: sdc-responses-dashboard-source
+    passed: [sdc-responses-dashboard-latest-deploy]
+    trigger: true
+  - get: ras-deploy
+  - task: run-unit-tests
```

Comment:
not sure we need to run the unit tests on a deploy into every env, if the unit tests have run and passed in the devtest deploy stage of the pipeline then wouldn't be enough?